app/src/main/java/de/tum/in/tumcampusapp/component/ui/search/adapter/RecentSearchesAdapter.kt

```diff
@@ -76,7 +75,7 @@ class RecentSearchesAdapter(
                         val lecture = Lecture.fromRecent(recentSearch)
                         titleText.text = lecture.title
                     } catch (exception: Exception) {
-                        titleText.setText(R.string.not_available_search)
+                        onRemove(recentSearch)
```

Comment:
I cannot reproduce to get an error from `fromRecent`. Is there a way this can fail that I am missing?
Can we just remove the try catch?

Reply:
I'm also looking for some way to reproduce it. 
If you think that would be better, we can do like that!

Reply:
Just removing the try catch would be good??